PROCESSES.md

```diff
@@ -27,13 +27,13 @@ description: DEPRECATED foo bar baz qux...
 
 ## Un-deprecating A Chart
 
-When new maintainers are interested in bring a chart out of deprecation with
+When new maintainers are interested in bring a chart out of depreciation with
 new features or new support that can be an option. To un-deprecate a chart:
 
 1. Update the maintainers on the chart if any are listed. The previous maintainers should not be expected to maintain the chart unless they explicitly decide to do so.
 1. If there is an OWNERS file in the chart that should be updated with the new reviewers and  approvers.
 1. The deprecated property from the `Chart.yaml` file should be removed along with any associated comment.
-1. The chart `version` needs to be incremented accordingly. If the same functionality is kept the version can be a patch increase. Otherwise the minor or major version needs to be incremented. For more detail on changing the version number see the [semver specification](http://semver.org).
+1. The chart `version` needs to be incremented accordingly. If the same functionality is kept the version can be a patch increase. Otherwise the minor or major version needs to be incremented. For more detail on changing the version number see the server specification](http://semver.org).
```

Comment:
The `[` in front of semver was to make it part of the link. Not it's a broken link. Can you undo this change?

Reply:
OK, I will change it.